.gitignore

```diff
@@ -0,0 +1,59 @@
+# Autosave files
```

Comment:
side question: where did you peeps found this template? I am going crazy with this stuff